src/components/forms/components/SearchAsYouTypeAjax.js

```diff
@@ -0,0 +1,699 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { _ } from 'underscore';
+import memoize from 'memoize-one';
+import ReactTooltip from 'react-tooltip';
+
+import { Fade } from './../../ui/Fade';
+
+import { ajax, object } from './../../util/';
+import { valueTransforms } from './../../util';
```

Comment:
These 2 lines can be combined into `import { ajax, object, valueTransforms } from './../../util/';`